dashboard/db/migrate/20170224150102_remove_activity_id_from_gallery_activities.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveActivityIdFromGalleryActivities < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
But since you already commented similarly...I'm wondering why you feel this way...